docs/user/dashboard/lens.asciidoc

```diff
@@ -240,7 +250,7 @@ image::images/lens_referenceLine_7.16.png[Lens drag and drop focus state]
 [[filter-the-data]]
 ==== Apply filters
```

Comment:
Something to think about for another PR.  Should we move the filter information to a general section about filtering in the concepts section?

Reply:
The filter functionality here is specific to Lens. If other Solutions/apps/etc pick up the Lens filter components, then yes!